src/Plugin/Field/FieldFormatter/AvPortalPhotoFormatter.php

```diff
@@ -170,11 +238,61 @@ protected function viewElement(FieldItemInterface $item): array {
       return [];
     }
 
-    return [
-      '#theme' => 'image',
+    $cache = new CacheableMetadata();
+
+    $image_style = $this->settings['image_style'];
+    $theme = $image_style !== "" ? 'image_style' : 'image';
+
+    $build = [
+      '#theme' => $theme,
       '#attributes' => ['class' => ['avportal-photo']],
-      '#uri' => $this->config->get('photos_base_uri') . $resource->getPhotoUri(),
+      '#uri' => 'avportal://' . $resource_ref . '.jpg',
```

Comment:
It needs to be added because this original stream URI is later down the line used to determine whether the images can be used in generating image styles. And if they dont have an extension of a an image that *can* be image style*d*, it won't work. We are "assuming" and/or enforcing that all images from the service are JPG. There is no way to tell that is what they all are but since they are photos, the odds are likely in that favor.